### PR TITLE
#5391 removed search results on map when identify tool selects a feature

### PR DESCRIPTION
**Pull request by alperr**
## Description
Search results are removed from map by firing a `cancelSelectedItem` action when a feature is selected.

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

Search results on map remain displayed after using identify tool to select another feature

**What is the new behavior?**
Search results are cleared when a feature is selected on map.

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

**Comment by alperr** [comment]
I just realized failing Travis-CI tests, I am fixing them now

File: web/client/epics/identify.js [hunk 1]
```diff
@@ -223,7 +224,7 @@ export default {
         })
             .switchMap(({point, layer}) => {
                 const projection = projectionSelector(store.getState());
-                return Rx.Observable.of(featureInfoClick(updatePointWithGeometricFilter(point, projection), layer))
+                return Rx.Observable.of(featureInfoClick(updatePointWithGeometricFilter(point, projection), layer), cancelSelectedItem())
```
Comment (MV88) [inline web/client/epics/identify.js:227]: can you add a unit test here to check that this action is dispatched as feature info click?

Comment (alperr) [reply web/client/epics/identify.js:227]: Hi Matteo, 

Isn't this test case below fulfill what you describe? 

https://github.com/geosolutions-it/MapStore2/blob/59715dfd851d713c5b9f3846f327eaa61c839ba2/web/client/epics/__tests__/identify-test.js#L558

I thought that this case was checking `if this action is dispatched as a feature info click`?

Please guide me if I misunderstood what you requested

Comment (MV88) [reply web/client/epics/identify.js:227]: if that test is the one affetecd by your changes then 
maybe you need to add a case check for the new action dispatched by increasing the number of actions expected.
if you need more help we can have a screenshare session

Comment (alperr) [reply web/client/epics/identify.js:227]: @MV88 I have updated the tests as you suggested

**Comment by MV88** [comment]
please solve conflict

**Review by MV88 — APPROVED** [review]

**Comment by MV88** [comment]
please @alperr backport this to 2020.02.xx

**Comment by tdipisa** [comment]
@alperr  the backport for this is still missing I think. Please provide it asap.

**Pull request merged**